svthalia/concrexit
# Add priority field to product list item

Closes #3419 



### Summary
Adds a priority field to `ProductListItem` that is ordered descendingly.

### How to test
Steps to test the changes you made:
1. Go to e.g. http://localhost:8000/api/v2/admin/sales/shifts/1/
2. See that the items are ordered by the priority entered in the admin
3. Try it in Thadmin and see that the same ordering is used there

TODO: maybe add the priority to the JSON?
